NeosModLoader/ModLoader.cs

```diff
@@ -10,7 +10,7 @@ namespace NeosModLoader
 {
 	public class ModLoader
 	{
-		internal const string VERSION_CONSTANT = "1.12.5";
+		internal const string VERSION_CONSTANT = "1.12.6";
```

Comment:
Is this really a patch change and not at least a minor one? ;'P

Reply:
My thoughts:
- From a mod dev perspective there's no API change.
- From a user perspective certain mods that used to crash in certain cases no longer crash

So even though the code change is insane, the impact is just like a bug fix. So I figure if it looks like a bug fix and it smells like a bug fix, then perhaps it *is* a bug fix.